language/stdlib/modules/DualAttestation.move

```diff
@@ -251,6 +251,7 @@ module DualAttestation {
         /// messages which fail verification and which do not.
         pragma opaque = true, verify = false;
         ensures result == spec_dual_attestation_message(payer, metadata, deposit_value);
+        aborts_if false;
```

Comment:
I probably would revert most or all of the spec changes, and just sync to the errors PR (which is now submitted). You may have an easier time reverting before rebase. As modifies is nowhere used in the library, thinks should just work.